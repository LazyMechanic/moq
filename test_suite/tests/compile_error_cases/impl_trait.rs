trait DummyTrait {}

#[moq::automock]
trait Trait {
    fn f(&self, _: impl DummyTrait);
}

fn main() {}
