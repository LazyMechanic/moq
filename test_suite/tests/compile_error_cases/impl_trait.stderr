error: `impl Trait` in argument position not supported, use generics `T: Trait` instead
 --> tests/compile_error_cases/impl_trait.rs:5:17
  |
5 |     fn f(&self, _: impl DummyTrait);
  |                 ^^^^^^^^^^^^^^^^^^
